deployment/README.md

```diff
@@ -0,0 +1,27 @@
+# Deployment
+
+This directory includes a shell script `setup.sh` and a few configuration files
+for packages that are required for hetmech backend deployment.
```

Comment:
One sentence per line.

Reply:
Don't you think that the line will be a little too long?